Sources/Plugins/ExternalSensorsPlugin/BLE/BLEManager/Sensors/BLEBikeSensor.swift

```diff
@@ -24,7 +24,15 @@ final class BLEBikeSensor: Sensor {
     var lastBikeCadenceData: BikeCadenceData?
     var lastBikeSpeedDistanceData: BikeSpeedDistanceData?
     // NOTE: wheelCircumference = wheelSize * pi
-    var wheelSize: Double = 2.086
+    var wheelSize: Double {
```

Comment:
Sensor class represents hardware sensor (reading/writing from/to device), it is not related to app settings in any way